elasticsearch/reindex/progress.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Usage: [watch -t] ./progress.sh <current index doc count>
+
+DOC_COUNT=$1
```

Comment:
But it's the old index that's being updated so on each request you will get the new count, which would make this more, or less, accurate dependant on how you want to look at it.

i.e.
If you have 100 images, start reindexing, we add 20 images and we get to 30/120 - 25%. But we are actually 30% on this step. But we are 25% complete of the whole process.